INSTALL.md

```diff
@@ -1,17 +1,55 @@
-# Install Zync (only explained for MacOS so far)
-1. Download this repository.
+# Install Zync
+
+## Download this repository.
 `git clone git@github.com:3scale/zync.git`
 
-2. Move to the folder of the project.
-`cd zync`
+## Setup PostgreSQL on Mac
+
+There is `Brewfile` containing all the dependencies.
+
+```
+cd zync
+brew bundle
+brew services start postgresql
+```
+
+The command `brew services start postgresql` starts the service of PostgreSQL. If later `./bin/setup` aborts, make sure that the `PostgreSQL` service is running. Verify with `brew services list` that has a status `started` and looking green. If the status is `started` but coloured orange, fix the errors indicated in the log located in `/usr/local/var/log/postgres.log`
+
+## Setup PostgreSQL on Fedora 34
 
-3. Install the dependencies. There is `Brewfile` containing all the dependencies.
-`brew bundle`
+```shell
+sudo dnf module install postgresql:10
+sudo dnf install libpq-devel
+sudo /usr/bin/postgresql-setup --initdb
+systemctl enable postgresql
+systemctl start postgresql
+psql -U postgres -c "create user ${USER}"
+psql -U postgres -c "create database zync_development with owner ${USER}"
+psql -U postgres -c "create database zync_test with owner ${USER}"
+psql -U postgres -c "create database zync_production with owner ${USER}"
+```
 
-4. Start postgres.
-`brew services start postgresql`
+## Setup PostgreSQL as a container with Docker or Podman
 
-5. Setup Zync.
+```
+docker run -d -p 5432:5432 -e POSTGRES_USER=postgres -e POSTGRES_DB=zync --name postgres10-zync docker.io/circleci/postgres:10.5-alpine
+```
+
+When running any Zync command, make sure to have `DATABASE_URL` environment variable set in prior.
```

Comment:
Why so? the `config/database.yml` is enough and it is Rails convention, no need to tell people to use that particularly over the config file

Reply:
My intention was to write a few commands that worked for me to setup the environment without having to figure it out each time. If you think there is no place for such instructions in the repo, I can remove and keep locally or put in porta-dev-tools or wherever.

Reply:
I think this is a common knowledge with Rails application. My remark is about the enforcing the user to use `DATABASE_URL` I do not think this should be the way because there is anotherway, editing`config/database.yml`

How about you write a wiki page "Quick start guide on Fedora 34" ? Or another Markdown file.

I would avoid to describe every cases in the README, it might get complicated to maintain it then.
My point is differentiating Zync purpose and other things.